include/deal2lkit/sacado_tools.h

```diff
@@ -31,6 +31,10 @@
 
 
 #include <Sacado.hpp>
+
+// Overload of std::sqrt, sts::log, std::sin, etc..
+#include <Sacado_ELRCacheFad_Ops.hpp>
```

Comment:
prima che lo aggiungessi non mi andava sqrt in piDomus.
Hai controllato? se i test passano anche senza a me va bene chiudere.


> On 25 May 2016, at 23:47, Alberto Sartori <notifications@github.com> wrote:
> 
> In include/deal2lkit/sacado_tools.h <https://github.com/mathLab/deal2lkit/pull/319#discussion_r64659299>:
> 
> > @@ -31,6 +31,10 @@
> >  
> >  
> >  #include <Sacado.hpp>
> > +
> > +// Overload of std::sqrt, sts::log, std::sin, etc..
> > +#include <Sacado_ELRCacheFad_Ops.hpp>
> questo header è già incluso dentro Sacado.hpp
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mathLab/deal2lkit/pull/319/files/a7b4806c36070c752a3cba585f64dd9b03086274#r64659299>